src/palace_tools/cli/patron_bookshelf.py

```diff
@@ -26,12 +26,8 @@ def main() -> None:
 )
 def patron_bookshelf(
     *,
-    username: str = typer.Argument(
-        None, help="Username to present to the OPDS server."
-    ),
-    password: str = typer.Argument(
-        None, help="Password to present to the OPDS server."
-    ),
+    username: str = typer.Option(..., "--username", "-u", help="Username or barcode."),
+    password: str = typer.Option(None, "--password", "-p", help="Password or PIN."),
```

Comment:
Updated these to match the `fetch_lcp` command, since one took args and one took options. Was a bit confusing when using the two commands back to back.